### ISSUE TITLE
Item model appears unshaded

### ISSUE THREAD

**Issue by IronPiston** [AtomicBlom/WeirdingGadget#23]
Forge: 34.1.19
Mod: 1.16.2-2.2.6

Item models have a darker shadow on the right face, but this effect is almost non-existent on the weirding gadget's item model.

![image](https://user-images.githubusercontent.com/11603684/95809101-94dbd600-0cd3-11eb-8744-e97731557f94.png)

**Comment by AtomicBlom** [comment]
Yeah, I'm losely aware of various shading issues with most of my mods for reasons I haven't yet nutted out.

It looks to me in that screenshot like the weirding gadget is actually shaded on the top and not the sides.
